index.js

```diff
@@ -5,11 +5,15 @@
  * @returns {boolean} whether the browser support text
  */
 function available() {
-  // https://gist.github.com/jonrohan/81085b119d16cdd7868a
```

Comment:
Have you checked out the implementation in `clipboard.js`? https://github.com/zenorocha/clipboard.js/blob/0c3bce265f6e8bbe6dfd63ec7b01c6e9394c8c93/src/clipboard.js#L87-L96 It might be more reliable than browser detection.

Reply:
Didn't feel all that great about these changes as I read through https://developer.mozilla.org/en-US/docs/Web/HTTP/Browser_detection_using_the_user_agent.

> It's worth re-iterating: it's very rarely a good idea to use user agent sniffing. You can almost always find a better, more broadly compatible way to solve your problem!

I can make another update this week. 😸